### DISCUSSION TITLE
Backup?

### DISCUSSION THREAD

**Discussion by msman0** [bassmaster187/TeslaLogger#1129] [Q&A]
Hi, wo kann man einstellen wie oft und wann das Backup gemacht wird?

**Comment by rowich** [comment]
Backups finde automatisch jede Nacht um 01:00 statt, dafür gibt es keine Einstellung. Dabei werden die privaten Geofence-Daten, die Logdatei (nur beim Monatswechsel) und die Datenbank gesichert. Am Ende dieses Prozesses wird noch die persönliche Datei (so sie existiert) "/etc/teslalogger/my-backup.sh" aufgerufen. Die könnte dazu benutzt werden, die entstandenen Daten auf ein anderes System, beispielsweise ein NAS, zu kopieren/verschieben.

Habe ich das nicht im Handbuch erwähnt? Wenn nicht: muss ich wohl nachführen. Das ist kein neuer Prozess.

**Reply by msman0** [reply]
Vielen Dank! Im Handbuch steht 1 mal pro Tag. Bei mir scheint der Automatismus nicht zu funktionieren. Ich habe es im Docker auf Synology installiert. Im Ordner Teslalogger/bin/backup landen nur die Dateien wenn ich manuell über das Webinterface ein Backup mache.
Geht der Automatismus in Docker nicht?

**Comment by rowich** [comment]
Oh, bei Docker und Synology bin ich überfragt. Da muss dann leider jemand anderes aushelfen. Ich fahre auf einem Raspberry PI 4 - und da funktioniert das so wie ich beschrieben habe.
@bassmaster187: Kein automatisches Backup im Docker-Umfeld?
